media/css/firefox/tiles.less

```diff
@@ -4,10 +4,135 @@
 
 @import "../sandstone/lib.less";
 
+#outer-wrapper {
+    overflow: hidden;
+}
+
 #wrapper {
     margin-top: 50px;
 }
 
-.sidebar .sidebar-inset img {
-    border: none;
+.sidebar {
+    float: left;
+    margin: -200px 10px 0 10px;
+
+    .sidebar-inset {
+        background-color: transparent;
+        box-shadow: none;
+        border: none;
+        padding: 0;
+
+        img {
+            border: none;
+            max-width: none;
+            margin-bottom: @gridGutterWidth;
+        }
+
+        img.tab {
+            width: 200%;
+        }
+
+        img.tab-preferences {
+            width: 100%;
+        }
+    }
 }
+
+.main-column {
+    width: 580px;
+
+    .intro {
+        .font-size(@baseLine);
+        font-family: "Open Sans Light";
```

Comment:
If you use `.open-sans-light;` here instead of directly using `font-family`, you'll get all the default fallback fonts as well as locale specific ones.